lib/i2cscan/i2cscan.cpp

```diff
@@ -113,13 +113,13 @@ namespace I2CSCAN
 
     bool hasDevOnBus(uint8_t addr) {
         byte error;
-#if ESP32C3
+#if defined(ESP32C3) || defined(ESP32C6)
```

Comment:
Was this neede for ESP32C6? (ESP32 does not need it as example)

Reply:
I will try